test/e2e/workload_identity_test.go

```diff
@@ -103,8 +103,8 @@ var _ = Describe("CSI inline volume test with workload identity", func() {
 	})
 
 	It("should read secret, key from pod", func() {
-		if !config.IsKindCluster {
-			Skip("test case currently supported for kind cluster only")
+		if !(config.IsKindCluster || config.IsSoakTest) {
```

Comment:
For now could we skip this test for Arc? The clusters created for conformance testing are not setup for WI, hence this test is failing.
We set `IsKindCluster=True` and `IsArcTest=true` while running conformance test